assets/js/components/product-attribute-control/index.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * External dependencies
+ */
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import apiFetch from '@wordpress/api-fetch';
+import { Component } from '@wordpress/element';
+import { find } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import SearchListControl from '../search-list-control';
+import SearchListItem from '../search-list-control/item';
+
+class ProductAttributeControl extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			list: [],
+			loading: true,
+		};
+	}
+
+	componentDidMount() {
+		apiFetch( {
+			path: addQueryArgs( '/wc-pb/v3/products/attributes', { per_page: -1 } ),
+		} )
```

Comment:
Since this'll eventually be merged into WC core, I don't want to make big architecture changes in the API– adding some properties seems fine, but I don't want to change how an entire endpoint works.

Reply:
Okay, that makes sense. 👍